src/style.css

```diff
@@ -0,0 +1,238 @@
+
+.sidebar__ {
```

Comment:
all the css for sidebar should be written in sidebar.css and Sidebar.js should import it.